PWGHF/D2H/TableProducer/dataCreatorDV0Reduced.cxx

```diff
@@ -0,0 +1,393 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file dataCreatorDV0Reduced.cxx
+/// \brief Creation of D-V0 pairs
+///
+/// \author Luca Aglietta <luca.aglietta@cern.ch>, UniTO Turin
+
+#include <cmath>
+#include <map>
+
+#include "DetectorsBase/Propagator.h"
+#include "CommonConstants/PhysicsConstants.h"
+#include "DCAFitter/DCAFitterN.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/O2DatabasePDGPlugin.h"
+#include "Framework/runDataProcessing.h"
+#include "ReconstructionDataFormats/DCA.h"
+
+#include "Common/Core/trackUtilities.h"
+#include "Common/DataModel/CollisionAssociationTables.h"
+
+#include "PWGLF/DataModel/LFStrangenessTables.h"
+
+#include "PWGHF/Core/HfHelper.h"
+#include "PWGHF/DataModel/CandidateReconstructionTables.h"
+#include "PWGHF/DataModel/CandidateSelectionTables.h"
+#include "PWGHF/Utils/utilsBfieldCCDB.h"
+#include "PWGHF/D2H/DataModel/ReducedDataModel.h"
+
+using namespace o2;
+using namespace o2::analysis;
+using namespace o2::aod;
+using namespace o2::constants::physics;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+// event types
+enum Event : uint8_t {
+  Processed = 0,
+  NoDV0Selected,
+  DV0Selected,
+  kNEvent
+};
+
+enum DecayChannel : uint8_t {
+  Ds1ToDstarK0s = 0,
+  DsStar2ToDplusK0s,
+  Xc3055ToDplusLambda
+};
+
+enum V0_type : int8_t {
```

Comment:
Not sure if this is needed anymore given the comment above, but in case you can modify it to 
```suggestion
enum TypeV0 : uint8_t {
```